sentinel/src/transitions/accept_patient_reports.js

```diff
@@ -114,6 +114,44 @@ const addRegistrationToDoc = (doc, registrations) => {
   }
 };
 
+const addReportUUIDToRegistration = (doc, registrations, callback) => {
```

Comment:
Are we sure that we're getting registrations in a specific order that means the most relevant one is first?

Are we sure that we can only have one registration out of the list that is valid to add a report UUID to?